src/main/java/com/passbook/sparkeighteen/peristence/repository/UserRepository.java

```diff
@@ -4,9 +4,12 @@
 import org.springframework.data.jpa.repository.JpaRepository;
 import org.springframework.stereotype.Repository;
 
+import java.math.BigInteger;
 import java.util.Optional;
 
 @Repository
 public interface UserRepository  extends JpaRepository<UserEntity, Integer> {
     Optional<UserEntity> findByEmail(String email);
```

Comment:
Enter missing

Reply:
Updated, As perviously discussed need this to be closed